internal/types/backup.go

```diff
@@ -111,6 +110,8 @@ type Operation interface {
 	SetState(s OperationState)
 	GetMessage() string
 	SetMessage(m string)
+	GetCompletedAt() *timestamppb.Timestamp
```

Comment:
As I remember, we wanted to keep this interface simple. The idea was to keep only methods needed to Operation Processor. Has something changed? @qrort

Reply:
I thought every operation type needs CompletedAt filled, so it is easier to do it with this interface. The alternative would be to switchcase over all operation types [here](https://github.com/ydb-platform/ydbcp/pull/40/files#diff-e9151bb5e1849c5c24847b82a3ab7b07a574834768b034d4ed6bb0cf70cf88d2R114)

Reply:
changed it to GetCompletedAt() to GetAudit(), i suggest to leave it in the interface, because it actually reduces boilerplate in `write.go` (there is more code in `backup.go` but we win overall)